src/common/transformers/TRUNCATE.spec.js

```diff
@@ -0,0 +1,13 @@
+import expect from 'expect';
+
+import { truncate } from './TRUNCATE';
+
+describe('TRUNCATE', () => {
+    it('#1', () => {
+        expect(truncate('hello world', 5)).toEqual('hello');
```

Comment:
So, the argument is the index (in the string/array) from where the values are dropped?

It could have been the number of values to drop (from the last one).